### PR TITLE
add varlociraptor estimate alignment properties

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/varlociraptor/estimatealignmentproperties/meta.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
```
Comment (sateeshperi) [inline modules/nf-core/varlociraptor/estimatealignmentproperties/meta.yml:2]:
```suggestion
```

File: modules/nf-core/varlociraptor/estimatealignmentproperties/main.nf [hunk 2]
```diff
@@ -0,0 +1,36 @@
+process VARLOCIRAPTOR_ESTIMATEALIGNMENTPROPERTIES {
+    tag "$meta.id"
+
```
Comment (sateeshperi) [inline modules/nf-core/varlociraptor/estimatealignmentproperties/main.nf:3]: missing label ?

Comment (FriederikeHanssen) [inline modules/nf-core/varlociraptor/estimatealignmentproperties/main.nf:3]:
```suggestion
    label 'process_single'
```

File: modules/nf-core/varlociraptor/estimatealignmentproperties/main.nf [hunk 3]
```diff
@@ -0,0 +1,36 @@
+process VARLOCIRAPTOR_ESTIMATEALIGNMENTPROPERTIES {
+    tag "$meta.id"
+    label 'process_single'
+    conda "bioconda::varlociraptor=8.1.1"
```
Comment (FriederikeHanssen) [inline modules/nf-core/varlociraptor/estimatealignmentproperties/main.nf:4]:
```suggestion
    
    conda "bioconda::varlociraptor=8.1.1"
```

**Comment by FriederikeHanssen** [comment]
Thnks @sateeshperi, fixed it

File: modules/nf-core/varlociraptor/estimatealignmentproperties/main.nf [hunk 4]
```diff
@@ -0,0 +1,37 @@
+process VARLOCIRAPTOR_ESTIMATEALIGNMENTPROPERTIES {
+    tag "$meta.id"
+    label 'process_single'
+
```
Comment (sateeshperi) [inline modules/nf-core/varlociraptor/estimatealignmentproperties/main.nf:4]:
```suggestion

```

Comment (sateeshperi) [reply modules/nf-core/varlociraptor/estimatealignmentproperties/main.nf:4]: trailing ws

**Review by sateeshperi — APPROVED** [review]

File: modules/nf-core/varlociraptor/estimatealignmentproperties/main.nf [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    """
+    varlociraptor estimate alignment-properties \\
+        $fasta \\
+        --bam $bam \\
+        $args \\
+        > ${prefix}.alignment-properties.json
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        : \$(echo \$(varlociraptor --version 2>&1) | sed 's/^.*varlociraptor //;' )
```
Comment (FriederikeHanssen) [inline modules/nf-core/varlociraptor/estimatealignmentproperties/main.nf:33]:
```suggestion
        : \$(echo \$(varlociraptor --version 2>&1) | sed 's/^.*varlociraptor //; s/:.*\$//' )
```

File: modules/nf-core/varlociraptor/estimatealignmentproperties/main.nf [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    """
+    varlociraptor estimate alignment-properties \\
+        $fasta \\
+        --bam $bam \\
+        $args \\
+        > ${prefix}.alignment-properties.json
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        : \$(echo \$(varlociraptor --version 2>&1) | sed 's/^.*varlociraptor //; s/:.*\$//' )
```
Comment (FriederikeHanssen) [inline modules/nf-core/varlociraptor/estimatealignmentproperties/main.nf:33]:
```suggestion
        varlociraptor: \$(echo \$(varlociraptor --version 2>&1) | sed 's/^.*varlociraptor //; s/:.*\$//' )
```

**Review by sateeshperi — APPROVED** [review]

**Pull request merged**